source/2019-05-25-the-ember-times-issue-100.md

```diff
@@ -32,12 +35,15 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Conditionally Wrapping Components with Ember.js 💝](https://dockyard.com/blog/2019/05/21/conditionally-wrapping-components-with-ember-js)
+
+Last year, we learned [how to yield an Ember component in multiple places](https://dockyard.com/blog/2018/11/26/how-to-yield-an-ember-component-in-multiple-places).
+
+This month, [Brandon Blaylock](https://github.com/baroquon) shares **step-by-step instructions** on how to extend this work so that [we can conditionally wrap components](https://dockyard.com/blog/2019/05/21/conditionally-wrapping-components-with-ember-js).
+
+The best news? With [angle bracket syntax](https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill), we can dramatically simplify our template! 💯
```

Comment:
```suggestion
The best news? With [angle bracket syntax](https://guides.emberjs.com/release/reference/syntax-conversion-guide/#toc_angle-bracket-syntax), we can dramatically simplify our template! 💯
```

Thoughts on using the guides or even the rfc?

Reply:
I'm ok with the link to our guide. I had linked to the polyfill so that more people could try out the angle bracket solution in the article.